<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>820</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="palFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenAs"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTileset">
    <property name="title">
     <string>Tileset</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionReportUse_Tileset"/>
    <addaction name="separator"/>
    <addaction name="actionCompressTileset_Tileset"/>
    <addaction name="actionResetFrameTypes_Tileset"/>
    <addaction name="separator"/>
    <addaction name="actionCleanupSubtiles_Tileset"/>
    <addaction name="actionCleanupFrames_Tileset"/>
    <addaction name="separator"/>
    <addaction name="actionSortSubtiles_Tileset"/>
    <addaction name="actionSortFrames_Tileset"/>
   </widget>
   <widget class="QMenu" name="menuPalette">
    <property name="title">
     <string>Palette</string>
    </property>
    <addaction name="actionNew_PAL"/>
    <addaction name="actionOpen_PAL"/>
    <addaction name="actionSave_PAL"/>
    <addaction name="actionSave_PAL_as"/>
    <addaction name="actionClose_PAL"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Translation_2"/>
    <addaction name="actionOpen_Translation_2"/>
    <addaction name="actionSave_Translation_2"/>
    <addaction name="actionSave_Translation_2_as"/>
    <addaction name="actionClose_Translation_2"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Translation_1"/>
    <addaction name="actionOpen_Translation_1"/>
    <addaction name="actionSave_Translation_1"/>
    <addaction name="actionSave_Translation_1_as"/>
    <addaction name="actionClose_Translation_1"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTileset"/>
   <addaction name="menuPalette"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenAs">
   <property name="text">
    <string>Open As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen_PAL">
   <property name="text">
    <string>Open palette...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOpen_Translation_1">
   <property name="text">
    <string>Open unique translation...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionOpen_Translation_2">
   <property name="text">
    <string>Open translation...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionReportUse_Tileset">
   <property name="text">
    <string>Usage Report</string>
   </property>
   <property name="toolTip">
    <string>List the uses of the current frame/subtile/tile</string>
   </property>
  </action>
  <action name="actionResetFrameTypes_Tileset">
   <property name="text">
    <string>Optimize Frames</string>
   </property>
   <property name="toolTip">
    <string>Use optimal compression type for all frames</string>
   </property>
  </action>
  <action name="actionCleanupFrames_Tileset">
   <property name="text">
    <string>Cleanup frames</string>
   </property>
   <property name="toolTip">
    <string>Delete frames not used by any subtile</string>
   </property>
  </action>
  <action name="actionCleanupSubtiles_Tileset">
   <property name="text">
    <string>Cleanup Subtiles</string>
   </property>
   <property name="toolTip">
    <string>Delete subtiles not used by any tile</string>
   </property>
  </action>
  <action name="actionCompressTileset_Tileset">
   <property name="text">
    <string>Deduplicate Tileset</string>
   </property>
   <property name="toolTip">
    <string>Deduplicate frames and subtiles</string>
   </property>
  </action>
  <action name="actionSortFrames_Tileset">
   <property name="text">
    <string>Sort Frames</string>
   </property>
   <property name="toolTip">
    <string>Sort the frames based on the subtiles</string>
   </property>
  </action>
  <action name="actionSortSubtiles_Tileset">
   <property name="text">
    <string>Sort Subtiles</string>
   </property>
   <property name="toolTip">
    <string>Sort the subtiles based on the tiles</string>
   </property>
  </action>
  <action name="actionSave_PAL">
   <property name="text">
    <string>Save palette</string>
   </property>
  </action>
  <action name="actionSave_PAL_as">
   <property name="text">
    <string>Save palette as...</string>
   </property>
  </action>
  <action name="actionSave_Translation_2">
   <property name="text">
    <string>Save translation</string>
   </property>
  </action>
  <action name="actionSave_Translation_2_as">
   <property name="text">
    <string>Save translation as...</string>
   </property>
  </action>
  <action name="actionSave_Translation_1">
   <property name="text">
    <string>Save unique translation</string>
   </property>
  </action>
  <action name="actionSave_Translation_1_as">
   <property name="text">
    <string>Save unique translation as...</string>
   </property>
  </action>
  <action name="actionNew_PAL">
   <property name="text">
    <string>New palette</string>
   </property>
  </action>
  <action name="actionClose_PAL">
   <property name="text">
    <string>Close palette</string>
   </property>
  </action>
  <action name="actionClose_Translation_2">
   <property name="text">
    <string>Close translation</string>
   </property>
  </action>
  <action name="actionClose_Translation_1">
   <property name="text">
    <string>Close unique translation</string>
   </property>
  </action>
  <action name="actionNew_Translation_2">
   <property name="text">
    <string>New translation</string>
   </property>
  </action>
  <action name="actionNew_Translation_1">
   <property name="text">
    <string>New unique translation</string>
   </property>
  </action>
  <action name="actionClone_Subtile">
   <property name="text">
    <string>Clone Subtile</string>
   </property>
   <property name="toolTip">
    <string>Clone current subtile</string>
   </property>
  </action>
  <action name="actionClone_Tile">
   <property name="text">
    <string>Clone Tile</string>
   </property>
   <property name="toolTip">
    <string>Clone current tile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
